spec/policies/course_lesson_part_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CourseLessonPartPolicy, type: :policy do
+  subject { described_class.new(user, course_lesson_part) }
```

Comment:
you can also add named subject for example subject(:foo) - just a thought

Reply:
Good to know :)

Reply:
👍